src/main/scanLogic/scanRunners/applicabilityScan.ts

```diff
@@ -268,21 +268,31 @@ export class ApplicabilityRunner extends JasRunner {
         let applicableCvesIdToDetails: Map<string, CveApplicableDetails> = new Map<string, CveApplicableDetails>(
             Object.entries(scanResponse.applicableCve)
         );
+        let nonapplicableCvesIdToDetails: Map<string, CveApplicableDetails> = new Map<string, CveApplicableDetails>(
```

Comment:
```suggestion
        let notApplicableCvesIdToDetails: Map<string, CveApplicableDetails> = new Map<string, CveApplicableDetails>(
```

Reply:
Took nonapplicableCve Name from line 55. 
Should I change it as well for a uniform code?

Reply:
yes